### ISSUE TITLE
event-scope false positives

### ISSUE THREAD

**Issue by ilyavolodin** [ilyavolodin/eslint-plugin-backbone#38]
event-scope incorrectly identifies an issue in the following code:

```js
this.$('.something').on('change', this.update);
```

Since this is jQuery event, and not Backbone's, third parameter suggestion is invalid. Code should be modified to check for `this.$().on\off`
